<launch>
   <node pkg="fiona_pkg" type="fiona_render_client" name="fiona_application_client" args="run" output="screen" />
</launch>
